gtests/net/mptcp/sockopts/mptcp_unsupported_sockopts_tcp_inq.pkt

```diff
@@ -0,0 +1,9 @@
+-tolerance_usecs=100000
+`../../common/defaults.sh`
+
+// test 1 : TCP_INQ is not supported by MPTCP sockets yet.
+// Follow https://github.com/multipath-tcp/mptcp_net-next/issues/224 for progress 
+// Once supported, we can replace it with gtests/net/mptcp_unsupported/mptcp_tcp_inq_client.pkt 
```

Comment:
this file doesn't exist. Did you forget to add it to git?

Reply:
> this file doesn't exist. Did you forget to add it to git?

Is the file missing or should we remove this line?

Reply:
@matttbe ah yes, I think we need to remove this for now because I don't have a test in  gtests/net/mptcp_unsupported/ to check for TCP_INQ when it's supported. Do you think it's a good idea to add the test or remove the line in question?

Reply:
The best would be to add a test but for now -- and if the main goal is to check EOPNOTSUPP is returned for unsupported sockopts -- we can simply remove this line.

Reply:
I guess the main goal was to check for EOPNOTSUPP. I'll go ahead and remove the line.